modules/Bio/EnsEMBL/Utils/IO/GFFSerializer.pm

```diff
@@ -60,25 +64,37 @@ my %strand_conversion = ( '1' => '+', '0' => '.', '-1' => '-');
 =head2 new
 
     Constructor
-    Arg [1]    : Ontology Adaptor
-    Arg [2]    : Optional File handle
-    Arg [3]    : Default source of the features. Defaults to .
-    
+    Arg [1]    : Optional File handle
+    Arg [2]    : Default source of the features. Defaults to .
+
     Returntype : Bio::EnsEMBL::Utils::IO::GFFSerializer
 
+    Warning    : If legacy mandatory 'Ontology Adaptor' first argument provided.
+                 This argument has been removed but method will continue functioning if still provided,
+                 raising only a warning to the user stating that it can be removed.
+
 =cut
 
 sub new {
     my $class = shift;
+    my $arg1 = shift;
+
+# Legacy constructor required an 'Ontology Adaptor' to be passed on as first argument
+# this is to support and keep scripts using the legacy constructor functioning (with an extra warning)
+# Can probably be removed some time in the future
+    if ( check_ref($arg1, "Bio::EnsEMBL::DBSQL::OntologyTermAdaptor" )) {
+        warning("GFF format does not require an instance of Bio::EnsEMBL::DBSQL::OntologyTermAdaptor anymore."
+            . "\nFirst argument at Bio::EnsEMBL::Utils::IO::GFFSerializer->new() can be safely removed.");
+        $arg1 = shift;
+    }
+
+    my $arg2 = shift;
+
     my $self = {
-        ontology_adaptor => shift,
-        filehandle => shift,
-        default_source => shift
+        filehandle => $arg1,
+        default_source => $arg2
```

Comment:
This is entirely cosmetic but it would be nice to have the fat commas vertically aligned here.

Reply:
done